anyway/infographics_utils.py

```diff
@@ -1238,7 +1335,7 @@ def get_latest_accident_date(table_obj, filters):
 def generate_widgets(request_params: RequestParams, to_cache: bool = True) -> List[Widget]:
     widgets = []
     # for w in WidgetId:
-    for w in WidgetCollection.get():
+    for w in get_widget_classes():
         # widget: Optional[Widget] = create_widget(w, request_params)
         widget: Optional[Widget] = w(request_params)
```

Comment:
When is this None? Seems like `w` is `Type[Widget]` meaning this call is a constructor call that never returns None. Is it possible for `w` to be a function?